### PR TITLE
Update js example to specify the cred file

### PR DESCRIPTION
**Pull request by zwmcoder**
## Description
As from example for js client lib:
https://github.com/nats-io/nats.js/blob/master/examples/nats-pub.js
And the connect ops definition:
https://github.com/nats-io/nats.js/blob/9d53eda057472aceb5b5af9451f2cb7a0d5d033b/index.d.ts#L54

The example here is out of date and not correct. Need update

### CODE REVIEW

**Comment by gcolliso** [comment]
@aricart could you review, please? Thank you!

File: developing-with-nats/security/creds.md [hunk 1]
```diff
@@ -50,11 +50,10 @@ nc.close();
 {% tab title="JavaScript" %}
 ```javascript
 // credentials file contains the JWT and the secret signing key
-let credsFile = path.join(confDir, 'credsfile.creds');
-
-let nc = NATS.connect({
-    url: server.nats,
-    userCreds: credsFile
+const data = fs.readFileSync(creds_file_path);
+  let nc = NATS.connect({
```
Comment (bruth) [inline developing-with-nats/security/creds.md:54]:
```suggestion
const data = fs.readFileSync(credsFile);
let nc = NATS.connect({
```

File: developing-with-nats/security/creds.md [hunk 2]
```diff
@@ -80,11 +79,11 @@ await nc.close()
 {% tab title="TypeScript" %}
 ```typescript
 // credentials file contains the JWT and the secret signing key
-let credsFile = path.join(confDir, 'credsfile.creds');
+const data = fs.readFileSync(creds_file_path);
```
Comment (bruth) [inline developing-with-nats/security/creds.md:82]:
```suggestion
const data = fs.readFileSync(credsFile);
```

**Review by bruth — CHANGES_REQUESTED** [review]
Minor code tweaks (naming and indent), but otherwise looks good.

File: developing-with-nats/security/creds.md (hunk 2 above)
Comment (aricart) [reply developing-with-nats/security/creds.md:82]: I would expand on the comment to say that reading of the credentials file will be different depending on the JavaScript runtime, and that the code shown there is for node.js

**Review by aricart — APPROVED** [review]
LGTM

File: developing-with-nats/security/creds.md [hunk 3]
```diff
@@ -80,11 +79,11 @@ await nc.close()
 {% tab title="TypeScript" %}
 ```typescript
 // credentials file contains the JWT and the secret signing key
```
Comment (bruth) [inline developing-with-nats/security/creds.md:81]:
```suggestion
// Credentials file contains the JWT and the secret signing key. This examples
// show how the file could be read using the Node.js runtime.
```

**Comment by bruth** [comment]
Merge the commits, but I now see there is a conflict since this is an older PR. Will look shortly.

**Comment by aricart** [comment]
@bruth  I want to also point out that these snippets which we copied right into the doc were actually managed, but when we did the change, we eliminated the CI checking on the sample code.

https://github.com/nats-io/node-nats-examples

**Pull request closed without merging**